org-cyf/content/databases/sprints/1/day-plan/index.md

```diff
@@ -17,9 +17,9 @@ time="60"
 name="Morning break"
 src="blocks/morning-break"
 [[blocks]]
-name="Writing bug reports"
-src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/reporting-bugs"
-time=60
+name="Full Stack: setting up, backing up, and restoring a database"
```

Comment:
Any idea why we get floating `>`s in the workshop? See screenshot:
<img width="958" alt="Screenshot 2024-04-11 at 00 07 24" src="https://github.com/CodeYourFuture/curriculum/assets/1131704/ba3ea295-2526-4cbe-ae03-0e9237d3459c">

Reply:
Don't think the day-plan view is parsing some of the markdown from the workshop properly. I'll raise a PR on the workshop repo.